agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -127,6 +127,45 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
+	stockpile_user=$(getconf.py stockpile_user stockpile)
+	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+	stockpile_log=$(getconf.py stockpile_log stockpile)
+
+	if [[ -z "$stockpile_path" ]]; then
+		stockpile_path=$(getconf.py pbench_install_dir DEFAULT)
+	fi
+	if [[ -d "$stockpile_path/stockpile" ]]; then
+        	rm -rf $stockpile_path/stockpile
+	fi
+
+	# set stockpile log file and output paths
+	if [[ -z "$stockpile_log" ]]; then
+                stockpile_log=$dir/stockpile.log
+        fi
+	if [[ -z "$stockpile_output_path" ]]; then
+        	stockpile_output_path=$dir/stockpile.json
+	fi
+
+	# generate inventory and run stockpile
+	generate_inventory stockpile > $INVENTORY
+	debug_log "[$script_name]Collecting stockpile data"
+	stockpile_opts="stockpile_user=$stockpile_user local_remote_user=$stockpile_local_remote_user host_remote_user=$stockpile_host_remote_user stockpile_output_path=$stockpile_output_path"
+	stockpile_playbook="$stockpile_path/stockpile/stockpile.yml"
+	echo "ansible-playbook -vv --extra-vars $stockpile_opts -i $INVENTORY $stockpile_playbook" >$stockpile_path/stockpile.cmd
```

Comment:
I think the quoting here is wonky: the `--extra-vars` argument is an unquoted space-separated list of `var=value` pairs. This shoulde be 

    echo "ansible-playbook -vv --extra-vars '"$stockpile_opts"' -i $INVENTORY $stockpile_playbook" >$stockpile_path/stockpile.cmd

Also, I don't think the  command should be in $stockpile_path, which if otherwise unspecified is `/opt/pbench-agent`: that should be considered read-only.

Reply:
Seems to have been resolved.